examples/movies/template.h0.html

```diff
@@ -158,9 +158,13 @@ <h3 class="movieTitle">Movie Title</h3>
       </template>
       <template id="listTemplate">
         <li>
-          <h3></h3>
-          <h4></h4>
-          <ul class="listMovies"></ul>
+          <a href="#">
+            <div class="listCard">
+              <h2></h2>
+              <span></span>
```

Comment:
Might be better to give it a class, like "listMovieCount". We use the tag names when it's clear-ish what they're for (e.g. headings, nav)